src/Security/ContaoAuthenticator.php

```diff
@@ -28,6 +30,21 @@
 class ContaoAuthenticator implements SimplePreAuthenticatorInterface
 {
     /**
+     * @var ContainerInterface
+     */
+    private $container;
+
+    /**
+     * Constructor.
+     *
+     * @param ContainerInterface $container The service container
+     */
+    public function __construct(ContainerInterface $container)
```

Comment:
This is an API change and cannot be done at the current point in time. We will have to extend `ContainerAware` instead.